docs/SplunkOperatorUpgrade.md

```diff
@@ -0,0 +1,77 @@
+​
+A Splunk Operator for Kubernetes upgrade might include support for a later version of the Splunk Enterprise Docker image. After the Splunk Operator completes its upgrade, the pods will be restarted using the latest Splunk Enterprise Docker image.
+​
+* Note: The Splunk Operator does not provide a way to downgrade to a previous release.
+​
+## Prequisites
+​
+* Before you upgrade, review the Splunk Operator [change log](https://github.com/splunk/splunk-operator/releases) page for information on changes made in the latest release. The Splunk Enterprise Docker image compatibility is noted in each release version.
+​
+* If the Splunk Enterprise Docker image changes, review the Splunk Enterprise [Upgrade Readme](https://docs.splunk.com/Documentation/Splunk/latest/Installation/AboutupgradingREADTHISFIRST) page before upgrading. 
+​
+* For general information about Splunk Enterprise compatibility and the upgrade process, see [How to upgrade Splunk Enterprise](https://docs.splunk.com/Documentation/Splunk/latest/Installation/HowtoupgradeSplunk).
+​
+* If you use forwarders, verify the Splunk Enterprise version compatibility with the forwarders in the [Compatibility between forwarders and Splunk Enterprise indexers](https://docs.splunk.com/Documentation/Forwarder/latest/Forwarder/Compatibilitybetweenforwardersandindexers) documentation.
+​
+## Upgrading Splunk Operator and Splunk Operator Deployment
+​
+1. Download the latest Splunk Operator installation yaml file.
+​
+```
+wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/0.2.2/splunk-operator-install.yaml 
+```
+​
+2. (Optional) Review the file and update it with your specific customizations used during your install. 
+​
+3. Upgrade the Splunk Operator.
+​
+```
+kubectl apply -f splunk-operator.yaml
+```
+​
+After applying the yaml, a new operator pod will be created and the existing operator pod will drop. Example:
+​
+```
+kubectl get pods
+NAME                               READY   STATUS    RESTARTS   AGE
+splunk-operator-75f5d4d85b-8pshn   1/1     Running   0          5s
+​
+```
+​
+If a Splunk Operator release changes the custom resource (CRD) API version, the administrator is responsible for updating their Custom Resource specification to reference the latest CRD API version.
+​
+If a Splunk Operator release includes an updated Splunk Enterprise Docker image, the operator upgrade will also initiate pod restart using the latest Splunk Enterprise Docker image.
+​
+## Verifiy Upgrade is Successful
+​
+To verify the Splunk Operator has been upgraded to the release image in splunk-operator-install.yaml, you can check the version of the operator image. Example:
```

Comment:
To verify the Splunk Operator has been upgraded to the desired version indicated in **`splunk-operator-install.yaml`**, you can check the version of the operator image in the deployment spec and subsequently the image in Pod spec of the newly deployed operator pod.